OSPSuite.Presentation.Importer/Presenters/ColumnMappingPresenter.cs

```diff
@@ -101,10 +102,8 @@ public void ChangeUnitsOnRow(ColumnMappingViewModel model)
             {
                column.Unit = unitsEditorPresenter.SelectedUnit;
                model.Description = ColumnMappingFormatter.Stringify(model.Source);
-               OnDataFormatParametersChanged?.Invoke(this, new DataFormatParametersChangedArgs()
-               {
-                  Parameters = new List<DataFormatParameter>() { activeRow.Source }
```

Comment:
IMO event parameters should only be readonly. This also allows to simplify the code by using a constructor